docs/source/actions.rst

```diff
@@ -196,13 +196,19 @@ List of recipients of the message. It could be a channel or an user.
 
 SPIDERMON_SLACK_SENDER_NAME
 ---------------------------
+Name of app/bot/user created for serving notifications on desired channels.
 
 .. _SPIDERMON_SLACK_SENDER_TOKEN:
 
 SPIDERMON_SLACK_SENDER_TOKEN
 ----------------------------
 
-Your `Slack token`_.
+Read `here`_ to know more about Slack Tokens.
+Get your Slack Credentials by `creating an app <https://api.slack.com/apps>`_  here.
```

Comment:
The documentation about `SPIDERMON_SLACK_SENDER_NAME` says ‘app/bot/user’, yet here you describe how users need to create a Slack ‘app’ to get things working. What is it? Can a regular user be used? Is there a difference between a bot and an app when it comes to using Slack from Spidermon?

You say ‘Slack credentials’ here. Does that mean just the token, or something more? If it’s something more that needs to be assigned to a setting, such as the value of `SPIDERMON_SLACK_SENDER_NAME`, maybe there should be a separate topic or section somewhere in the documentation, if there is nothing yet, that covers setting up Slack and explains the required steps to create an app and then fill the required settings in Spidermon, and have the documentation of these settings be minimalistic and simply link to that topic for users needed more information.

Some :lipstick: aesthetic notes:
- I think we should be consistent with vocabulary: if users need to create an app, lets tack about Slack apps (neither bots nor users).
- To make a new paragraph, leave a black line
- I think you can remove the ‘here’ here :slightly_smiling_face:

Reply:
You are right, it does sound confusing. As I mentioned before, Slack as both `Slack Apps` which with added functionality would become bots. I think we should go with Slack Apps to keep it simple. I agree on the part of setting up a separate minimalistic guide of setting up the Slack App. 

There is only one token needed, I will fix the `Slack Credentials` and other suggestions as well.